examples/src/main/scala/example/calibantotapir/tapir/GraphiqlEndpoint.scala

```diff
@@ -0,0 +1,82 @@
+package example.calibantotapir.tapir
+
+import sttp.tapir.ztapir._
+import zio._
+import sttp.capabilities.zio.ZioStreams
+
+object GraphiQLEndpoint {
+  val graphiql: ZServerEndpoint[Any, ZioStreams] =
+    infallibleEndpoint.get
+      .in("graphiql")
+      .out(htmlBodyUtf8)
+      .serverLogic(_ => ZIO.right(html(apiPath = "graphql", uiPath = "graphiql")))
+
+  private def html(apiPath: String, uiPath: String): String =
```

Comment:
Is this the same html that's served via the `quick` module? If yes, should we move it in a common module so that it can be used by the tapir modules as well?

Reply:
yessir! it's the exact same

Reply:
Should we put it in tapir-interop? Looks like quick can access it then